changelog/7360.bugfix.rst

```diff
@@ -0,0 +1 @@
+Disable caching when evaluating expressions in marks.
```

Comment:
The changelog should describe the bug this is fixed, rather than the fix itself, which is an internal/technical detail.

In this case I would write something like this:

```suggestion
Fix possibly incorrect evaluation of string expressions passed to ``pytest.mark.skipif` and ``pytest.mark.xfail``,
in rare circumstances where the exact same string is used but refers to different global values.
```